pkg/distributor/distributor_test.go

```diff
@@ -426,7 +494,7 @@ func TestDistributor_PushIngestionRateLimiter(t *testing.T) {
 		ingestionBurstSize int
 		pushes             []testPush
 	}{
-		"limit should be evenly shared across distributors": {
+		"ingestion limit should be evenly shared across distributors": {
```

Comment:
```suggestion
		"Evenly share the ingestion limit across distributors": {
```
Work around the term _should_, which can be ambiguous. I have provided an alternative.